web/src/app/util/Markdown.js

```diff
@@ -16,6 +16,12 @@ export default function Markdown(props) {
         if (node.type !== 'link') return true
         if (node.children[0].type !== 'text') return true // only validate text labels
         if (safeURL(node.url, node.children[0].value)) return true
+        if (node.url.startsWith('mailto:')) {
+          // do not yield native mailto link, use plain text instead
+          node.type = 'text'
+          delete node.url
+          return true
+        }
```

Comment:
Not sure it's necessary to bar the links everywhere in-app -- this will also prevent `mailto` links from working in descriptions and alert details when added intentionally by users of the application.

I think the goal was only to fix it on the user details page by making it render as text.

Reply:
For profile maybe something like `noMarkdown` as a flag for `DetailsPage`

As for the others, I wasn't aware that other protocols were broken, that seems like a bug. As far as URLs that are not allowed, it was intentional to render them as raw markdown rather than hide any information.

The primary concern was labels that don't match the URL. In that case a fix could be to allow the label to omit the protocol in the URL so that things like both `[http://example.com](http://example.com)` and `[example.com](http://example.com)` work